symfony/plugins/orangehrmLeavePlugin/Api/EmployeeLeaveRequestAPI.php

```diff
@@ -43,6 +43,9 @@
 use OrangeHRM\Leave\Api\Traits\LeaveRequestPermissionTrait;
 use OrangeHRM\Leave\Dto\LeaveRequest\DetailedLeaveRequest;
 use OrangeHRM\Leave\Dto\LeaveRequestSearchFilterParams;
+use OrangeHRM\Leave\Exception\LeaveAllocationServiceException;
+use OrangeHRM\Leave\Service\LeaveApplicationService;
```

Comment:
if this import not in use, please remove it

Reply:
done